tests/mappers/test_mane_transcript.py

```diff
@@ -420,21 +417,18 @@ async def test_c_to_mane_c(test_mane_transcript, braf_v600e_mane_c, egfr_l858r_m
         "NM_004448.3", 2264, 2278, AnnotationLayer.CDNA, ref="TGAGGGAAAACACAT"
     )
     assert mane_c == CdnaRepresentation(
-        **{
-            "alt_ac": "NC_000017.11",
-            "refseq": "NM_004448.4",
-            "ensembl": "ENST00000269571.10",
-            "pos": (2263, 2278),
-            "status": TranscriptPriority.MANE_SELECT,
-            "strand": Strand.POSITIVE,
-            "coding_start_site": 175,
-            "coding_end_site": 3943,
-            "gene": "ERBB2",
-        }
+        alt_ac="NC_000017.11",
+        refseq="NM_004448.4",
+        ensembl="ENST00000269571.10",
+        pos=(2263, 2278),
+        status=TranscriptPriority.MANE_SELECT,
+        strand=Strand.POSITIVE,
+        coding_start_site=175,
+        coding_end_site=3943,
+        gene="ERBB2",
     )
 
 
-@pytest.fixture(scope="function")
```

Comment:
@korikuzma I can't quite tell what's going on here -- this is tagged as a `fixture` but includes assertions, so I removed the fixture tag... and that caused four or five test cases to break :(

Reply:
@jsstevenson I think this is what we want:

```python
@patch.object(SeqRepoAccess, "get_reference_sequence")
def test_get_prioritized_transcripts_from_gene(test_get_seqrepo, test_mane_transcript):
    """Test that _get_prioritized_transcripts_from_gene works as expected"""

    def get_reference_sequence(ac):
        """Return test response when getting sequence for a given accession"""
        dummy_tx_ref_seq = {
            "NM_004333.6": ("AC", None),
            "NM_004333.5": ("ACTG", None),
            "NM_001378472.1": ("A", None),
            "NM_001374258.2": ("A", None),
        }
        return dummy_tx_ref_seq[ac]

    test_get_seqrepo.return_value = None
    test_mane_transcript.seqrepo_access.get_reference_sequence = get_reference_sequence

    # cds_start_i does not impact this method
    data = [
        ["NP_004324.2", "NM_004333.6", "NC_000007.13", 0],
        ["NP_004324.2", "NM_004333.5", "NC_000007.13", 0],
        ["NP_001365401.1", "NM_001378472.1", "NC_000007.13", 0],
        ["NP_001365401.1", "NM_001374258.2", "NC_000007.13", 0],
        ["NP_004324.2", "NM_004333.6", "NC_000007.14", 0],
        ["NP_004324.2", "NM_004333.5", "NC_000007.14", 0],
        ["NP_001365401.1", "NM_001378472.1", "NC_000007.14", 0],
        ["NP_001365401.1", "NM_001374258.2", "NC_000007.14", 0],
    ]
    test_df = pl.DataFrame(data, schema=["pro_ac", "tx_ac", "alt_ac", "cds_start_i"])

    resp = test_mane_transcript._get_prioritized_transcripts_from_gene(test_df)
    assert resp == ["NM_004333.6", "NM_001374258.2", "NM_001378472.1"]

```